File: front/src/app/core/domain/message/game-created.message.ts
import {GameDto} from "@domain/games/game.dto";
import { BaseMessage } from "./base.message";
import { MessageTypeEnum } from "./message-type.enum";

export class GameCreatedMessage extends BaseMessage {
  override readonly type = MessageTypeEnum.GAME_CREATED;
  game: GameDto;
}

File: back/src/main/java/fr/uha/ensilan/concours/back/dto/message/GameCreatedMessage.java
package fr.uha.ensilan.concours.back.dto.message;

import fr.uha.ensilan.concours.back.dto.game.GameDto;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class GameCreatedMessage extends BaseMessage {
    private GameDto game;

    public GameCreatedMessage() {
        super(MessageType.GAME_CREATED);
    }
}
